README.md

```diff
@@ -27,143 +27,124 @@ The diagram below illustrates high-level design of each component as well as bas
 
 ![Architecture diagram](./assets/architecture.png)
 
-## Usage
+## Installation
+
+Official releases are available as debian packages which can be found under our [releases](https://github.com/0xPolygonMiden/miden-node/releases) page.
+
+Alternatively, the Rust package manager `cargo` can be used to install on non-debian distributions or to compile from source.
 
-Before you can build and run the Miden node or any of its components, you'll need to make sure you have Rust [installed](https://www.rust-lang.org/tools/install). Miden node v0.2 requires Rust version **1.78** or later.
+### Intall using `cargo`
```

Comment:
Before this section, should we add a short "Installing debian package"? It could a big part of the old "Debian Packages" section.